src/EnrollCallAndSmsController.js

```diff
@@ -90,6 +97,8 @@ function (Okta, FormController, Footer, PhoneTextBox, TextBox, CountryUtil, Form
         }
       },
       limitResending: function () {
+        this.set({timeoutWarning: true});
```

Comment:
since we show the warning callout at the same time as we enable the button with `Re-send`, can we use the same var we already have `ableToResend` instead of creating a new one? Or was there any reason to create a new var?

Reply:
make sense, I can fix it. I used the variable with the intention to indicate "true" to show a warning and "false" to disable the warning, boolean values are opposite to `ableToSend,` however, I am listening to the events with a CHANGE of the value, the logic should be the same, totally agreed.